plasma_framework/test/src/framework/ExitGameController.test.js

```diff
@@ -417,53 +416,6 @@ contract('ExitGameController', () => {
                 });
             });
         });
-
-        describe('When reentrancy happens', () => {
```

Comment:
Can we update to the present the new protection schema?

Reply:
done